README.md

```diff
@@ -4,14 +4,26 @@
 
 A streaming compression/decompression library for rust with bindings to libbz2.
 
-```toml
-# Cargo.toml
-[dependencies]
-bzip2 = "0.4"
+## Features
+
+By default, this crate will compile the `libbz2` C library from source. That requires a functional C toolchain,
+and may not work for all targets (in particular webassembly).
+
+*`libbz2-rs-sys`*
+
+Since version 0.5.0, this crate also supports using [libbz2-rs-sys](https://crates.io/crates/libbz2-rs-sys),
```

Comment:
This should probably be kept out until right before 0.5 is actually released to avoid confusing people who are looking at the readme on github rather than on crates.io.

Reply:
sure, to refer back to this later, the proposed text for the release is

> ## Features
> 
> By default, this crate will compile the `libbz2` C library from source. That requires a functional C toolchain,
> and may not work for all targets (in particular webassembly).
> 
> *`libbz2-rs-sys`*
> 
> Since version 0.5.0, this crate also supports using [libbz2-rs-sys](https://crates.io/crates/libbz2-rs-sys),
> a drop-in compatible rust implementation of `libbz2`. With this feature enabled, cross-compilation should work
> like any other rust code, and no C toolchain is needed to compile this crate or its dependencies.
> 
> ```sh
> bzip2 = { version = "0.5.0", default-features = false, features = ["libbz2-rs-sys"] }
> ```
> 
> *`static`*
> 
> Statically links `libbz2`. When `libbz2-rs-sys` is enabled, static mode is always used.